src/client/components/pages/privacy.js

```diff
@@ -20,6 +20,10 @@ const React = require('react');
 
 const PageHeader = require('react-bootstrap').PageHeader;
 
+/**
+* Renders the document and displays the 'privacy policy of BookBrainz'
```

Comment:
For this comment, I'd say something like:

    Renders a page containing a link to the current MusicBrainz privacy policy,
    which will form the basis of a future privacy policy for BookBrainz.